__test__/getBlogs.test.js

```diff
@@ -0,0 +1,65 @@
+const request = require('supertest');
+
+const app = require('../server/app');
+const { runBuild } = require('../server/database/config/build');
+const connection = require('../server/database/config/connection');
+const { getBlogs } = require('../server/database/queries');
+
+describe('Test getBlogs DB queries and route', () => {
+  beforeEach(() => runBuild());
+  afterAll(() => connection.end());
+  afterAll(async () => {
+    await new Promise((resolve) => setTimeout(() => resolve(), 3000));
+  });
```

Comment:
suggestion, you can delete this line because we use `--runInBand` flag in the test script to solve this problem